lib/shell/run_python.sh

```diff
@@ -2,27 +2,67 @@
 
 unset run_python
 run_python () {
-
+   trap - ERR # allow internal error handling
+   
     # get arguments
     program="$1"
     logfile="$2"
+    OUTPUT_DIR=$(dirname "$logfile")
 
     # set python command if unset
     if [ -z "$pythonCmd" ]; then
```

Comment:
Thanks @ShiqiYang2022 , this would be handled [here](https://github.com/gentzkow/GentzkowLabTemplate/blob/f263307512aa6bd76a2f822cef933898ebc9af61/lib/shell/run_python.sh#L18). Let me know if you disagree with this approach.

Reply:
That makes sense, thanks for the clarification!